package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "@scalar-labs/scalardl-javascript-sdk-base": "^1.3.1",
+    "@scalar-labs/scalardl-javascript-sdk-base": "^2.0.1",
+    "arraybuffer-loader": "^1.0.8",
```

Comment:
I am not sure but is this only used in the integration test ?

Reply:
Okay i will move it to dev